mu4e expects msg2pdf in some strange path, fix that
also remove the reference to the build path
See bug #760598
---
 mu4e/mu4e-actions.el |    2 +-
 mu4e/mu4e-meta.el.in |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- maildir-utils.git.orig/mu4e/mu4e-actions.el
+++ maildir-utils.git/mu4e/mu4e-actions.el
@@ -49,7 +49,7 @@
 
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-(defvar mu4e-msg2pdf (concat mu4e-builddir "/toys/msg2pdf/msg2pdf")
+(defvar mu4e-msg2pdf "/usr/bin/msg2pdf"
   "Path to the msg2pdf toy.")
 
 (defun mu4e-action-view-as-pdf (msg)
--- maildir-utils.git.orig/mu4e/mu4e-meta.el.in
+++ maildir-utils.git/mu4e/mu4e-meta.el.in
@@ -2,7 +2,7 @@
 (defconst mu4e-mu-version "@VERSION@"
   "Required mu binary version; mu4e's version must agree with this.")
 
-(defconst mu4e-builddir "@abs_top_builddir@"
+(defconst mu4e-builddir "IGNORED"
   "Top-level build directory.")
 
 (defconst mu4e-doc-dir "@MU_DOC_DIR@"
